environment_variables.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+CONSUMER_KEY = os.getenv("CONSUMER_KEY")  # twitter
```

Comment:
CONSUMER_KEY -> TWITTER_CONSUMER_KEY, etc. and remove comments

Reply:
Fixed, thank you!